### PR TITLE
Relax checking of state updates [KVL-848]

### PR DESCRIPTION
**Pull request by miklos-da**
This PR relaxes checking of `TransactionAccepted` state updates in the following ways by the integrity-checker:
* *Blinding info is not checked.* We have started adding blinding info to transaction updates only recently, hence, it is not present in older ledger exports.
* *Transaction ID is not checked.* Transaction ID is generated from the serialized form of a Daml submission in case of batched submissions which is not expected to stay the same across Daml SDK versions.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by fabiotudone-da — APPROVED** [review]
I'd only add the explanation in the PR as comments in the code.

**Comment by miklos-da** [comment]
> I'd only add the explanation in the PR as comments in the code.

Good point -- commented on `ReadServiceStateUpdateComparison.normalizeUpdate`.

**Review by SamirTalwar-DA — COMMENTED** [review]
It looks good to me.

Should we make these optional flags so we _do_ check these on versions which we know contain the relevant information? Perhaps something like `--as-of=YYYY-MM-DD`, and then compare the date against a range for each feature.

**Comment by miklos-da** [comment]
> It looks good to me.
> 
> Should we make these optional flags so we _do_ check these on versions which we know contain the relevant information? Perhaps something like `--as-of=YYYY-MM-DD`, and then compare the date against a range for each feature.

@SamirTalwar-DA Added some knobs so that the comparison can be controlled in a fine-grained way (to be wired in in a follow-up PR). Please let me know if you have concerns with this direction.

File: ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/StateUpdateComparison.scala [hunk 1]
```diff
@@ -56,6 +56,17 @@ final class ReadServiceStateUpdateComparison(
...
+
+  case class NormalizationSettings(
+      ignoreBlindingInfo: Boolean = false,
+      ignoreTransactionId: Boolean = false,
+  )
+
+  val DefaultNormalizationSettings: NormalizationSettings = NormalizationSettings(
+    ignoreBlindingInfo = true,
+    ignoreTransactionId = true,
+  )
```
Comment (SamirTalwar-DA) [inline ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/StateUpdateComparison.scala:68]: Seems sensible. Happy for this to be configurable in another changeset.

**Review by SamirTalwar-DA — APPROVED** [review]
